assets/styles.css

```diff
@@ -0,0 +1,842 @@
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+  font-family: 'Times New Roman', Times, serif;
+}
+
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
+body {
+  line-height: 1;
+  width: 100%;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::before,
+blockquote::after,
+q::before,
+q::after {
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+/* NAV BAR */
+
+nav {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  background-color: #fff;
+  padding: 0.5em 0;
+  border-bottom: 1px solid #e2e2e2;
+}
+
+/* TOP LEFT */
+.top-left {
+  display: flex;
+  width: 200px;
+  justify-content: space-between;
+}
+
+aside > span > a {
+  text-decoration: none;
+  font-size: 14px;
+  line-height: 17px;
+  color: #515151;
+  padding-top: 1em;
+  padding-left: 00.5em;
+  font-weight: 500;
+}
+
+.writer a {
+  text-decoration: none;
+  color: #515151;
+}
+
+.top-left a {
+  text-decoration: none;
+  color: #000;
+  font-weight: bold;
+}
+
+.top-left > a:hover {
+  text-decoration: underline;
+}
+
+/* TOP RIGHT BUTTONS */
+
+.top-right-btns button {
+  background-color: rgb(86, 123, 149);
+  border: 1px solid rgb(50, 104, 145);
+  font-weight: 700;
+  font-size: 11px;
+  line-height: 11px;
+  padding: 8px;
+  border-radius: 0.3em;
+  color: #fff;
+}
+
+/* MAIN ARTICLE */
+
+main {
+  width: 100%;
+}
+
+h1 {
+  font-family: 'PT Serif', serif;
+  font-size: 2.2rem;
+  line-height: 3rem;
+  font-style: italic;
+  font-weight: 500;
+  margin-top: 1.5em;
+  margin-bottom: 0.8em;
+  width: 600px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+b {
+  font-weight: 700;
+}
+
+figure {
+  width: 65%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+main img {
+  display: block;
+  margin: 0 auto;
+  width: 100%;
+}
+
+main figcaption {
+  color: #666;
+  font-size: 0.9375rem;
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+}
```

Comment:
- [ ] Change these inefficient selectors. and others found in the `style.css` file for better and more specific class selectors.